### PR TITLE
Add embargoReleaseDate to the show route

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

So we have a way of getting the embargo date for the object

## Was the API documentation (openapi.json) updated?

yes

### CODE REVIEW

**Comment by peetucket** [comment]
Looks like a test failure (need to check to be sure that embargoMetadata is even present as it won't be for collections)...

**Comment by jcoyne** [comment]
@peetucket okay, I've added a check for that.

File: spec/requests/show_object_spec.rb [hunk 1]
```diff
@@ -22,4 +22,20 @@
...
+    before do
+      object.descMetadata.title_info.main_title = 'Hello'
+      object.label = 'foo'
+      object.embargoMetadata.release_date = DateTime.parse '2019-09-26T07:00:00Z'
+    end
+
+    it 'returns the object' do
+      get '/v1/objects/druid:mk420bs7601',
+          headers: { 'Authorization' => "Bearer #{jwt}" }
+      expect(response).to be_successful
```
Comment (mjgiarlo) [inline spec/requests/show_object_spec.rb]: Should this be `.to have_http_status(:ok)` instead?

Comment (jcoyne) [reply spec/requests/show_object_spec.rb]: You're right that that would be more specific

**Pull request merged**